packages/serverless-workflow-language-service/src/channel/customValidations.ts

```diff
@@ -0,0 +1,125 @@
+import * as jsonc from "jsonc-parser";
+import { sourcePaths, targetPaths } from "./validationPaths";
+import { findNodesAtLocation, SwfLsNode } from "./SwfLanguageUtilMethods";
+import { Position, TextDocument } from "vscode-json-languageservice";
+
+export type TargetPathType = {
+  path: string[];
+  type: string;
```

Comment:
Maybe we can improve this type. Not all strings are allowed, so something like this would be better.

```suggestion
  type: "string" | "string[]" | "number" | "boolean";
```